src/arutils.py

```diff
@@ -239,12 +258,69 @@ def func_fit(x, y, func, deg, minv=None, maxv=None, w=None, **kwargs):
         return np.polynomial.chebyshev.chebfit(xv, y, deg, w=w)
     elif func == "bspline":
         return bspline_fit(x, y, order=deg, w=w, **kwargs)
+    elif func in ["gaussian"]:
+        # Guesses
+        if guesses is None:
+            mx = np.max(y)  # Could take simple stats near center
```

Comment:
Should this be:  `np.max(y)-max(0.0,np.min(y))`  (i.e. to estimate the _amplitude_)